.travis.yml

```diff
@@ -39,6 +39,7 @@ before_script:
   - rustup component add rust-src
   - rustup component add clippy
   - rustup component add rustfmt
+  - cargo install grcov
```

Comment:
We also need to install it by `.github/workflows/check.yml` in `Set up Rust` step to run these tests on github actions as well

Reply:
And looks like we also need to install nightly toolchain because we always use `+nightly` channel for coverage
But I'd prefer to teach our runner to use default nightly toolchain if possible - #5395